streams/src/test/java/org/apache/kafka/streams/integration/EOSUncleanShutdownIntegrationTest.java

```diff
@@ -73,16 +75,25 @@ public class EOSUncleanShutdownIntegrationTest {
 
     @SuppressWarnings("deprecation")
     @Parameterized.Parameters(name = "{0}")
-    public static Collection<String[]> data() {
-        return Arrays.asList(new String[][] {
-            {StreamsConfig.EXACTLY_ONCE},
-            {StreamsConfig.EXACTLY_ONCE_V2}
-        });
+    public static Collection<Object[]> data() {
+        final List<String> eosConfigs = Arrays.asList(
+            StreamsConfig.EXACTLY_ONCE,
+            StreamsConfig.EXACTLY_ONCE_V2
+        );
+        final List<Object[]> data = new ArrayList<>(eosConfigs.size() * 2);
+        for (final String eosConfig : eosConfigs) {
+            //data.add(new Object[] {eosConfig, false});
+            data.add(new Object[] {eosConfig, true});
```

Comment:
Looks like you intended for this test to be parameterized for both transactional and non-transactional stores, but have not yet completed it?